tox.ini

```diff
@@ -31,6 +31,19 @@ commands =
     python --version
     pytest -p no:warnings --cov=baybe --durations=5 {posargs}
 
+[testenv:singlepointprecision,singlepointprecision-py{310,311,312}]
+description = Run specific tests with single point precision
+extras = test
+passenv =
+    CI
+setenv =
+    BAYBE_NUMPY_USE_SINGLE_PRECISION = true
+    BAYBE_TORCH_USE_SINGLE_PRECISION = true
+    BAYBE_TEST_ENV = SINGLEPOINT
```

Comment:
does this come from copypasta? I don't think its needed or has any function

Reply:
Can you elaborate? By my understanding, this defines a new `tox` command which will use single-point precision and has a specific test name, so I do not get what you mean here.

Reply:
im asking about the point of this particular line `BAYBE_TEST_ENV = SINGLEPOINT`
What does it do?

Theres an equivalent line in another tox command, where it has a specific purpose. I do not see that purpose here.

Reply:
Yes, then this can probably go, you are right.

Reply:
Have removed that part, hence resolving here.